.idea/modules.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
shall we git ignore idea files since it's editor related

Reply:
It probably makes sense to use the gitignore file of another project like Vue as a template.